it/controllers/TooManyResultsISpec.scala

```diff
@@ -0,0 +1,203 @@
+package controllers
+
+import itutil.config.AddressRecordConstants._
+import itutil.config.IntegrationTestConstants._
+import itutil.{IntegrationSpecBase, PageContentHelper}
+import model._
+import play.api.http.HeaderNames
+import play.api.http.Status._
+import play.api.libs.json.{JsArray, JsObject, Json}
+import play.api.test.FakeApplication
+import uk.gov.hmrc.address.v2.Country
+
+class TooManyResultsISpec extends IntegrationSpecBase with PageContentHelper {
+
+  override implicit lazy val app = FakeApplication(additionalConfiguration = fakeConfig())
+
+  object tooManyResultsMessages {
+    val title = "No results found"
+    val heading1 = "There are too many results"
+    val heading2 = "We couldn't find any results for that property name or number"
+
+    def bullet1(postcode: String) = s"$postcode for postcode"
+
+    val bullet2NoFilter = "nothing for property name or number"
+
+    def bullet2WithFilter(filter: String) = s"'$filter' for name or number"
+
+    val line1 = "You entered:"
+    val button = "Try a new search"
+  }
+
+  object otherPageMessages {
+    val noResultsPageTitle = "Can't find any addresses"
+
+  }
+
+  "The 'Too Many Results' page" should {
+    "be rendered" when {
+      "the back buttons are enabled in the journey config" when {
+        "no filter has been entered" when {
+          "the backend service returns too many addresses" in {
+            stubKeystore(testJourneyId, testConfigDefaultWithResultsLimitAsJson, OK)
+            stubGetAddressFromBE(addressJson = addressRecordJsonList(numberOfRepeats = 51))
+
+            val res = buildClientLookupAddress(path = "select?postcode=AB11+1AB&filter=")
+              .withHeaders(HeaderNames.COOKIE -> sessionCookieWithCSRF, "Csrf-Token" -> "nocheck")
+              .get()
+
+            await(res).status shouldBe OK
+
+            val doc = getDocFromResponse(res)
+
+            doc.select("a[class=back-link]") should have(
+              text("Back")
+            )
+            doc.title shouldBe tooManyResultsMessages.title
+            doc.h1.text shouldBe tooManyResultsMessages.heading1
+            doc.select("t").text shouldBe tooManyResultsMessages.line1
```

Comment:
"t" isn't a valid HTML tag. Please update to a "p" tag in the view, then use:

`doc.paras should not have elementWithValue(tooManyResults.line1)`